tests/src/test/scala/com/nvidia/spark/rapids/RegularExpressionTranspilerSuite.scala

```diff
@@ -189,6 +181,12 @@ class RegularExpressionTranspilerSuite extends FunSuite with Arm {
     assertCpuGpuMatchesRegexpFind(patterns, Seq("", "\u0007", "a\u0007b", 
         "\u0007\u003f\u007f", "\u007f", "\u007f2"))
   }
+
+  test("hex digits >= 0x7F - find") {
+    val patterns = Seq(raw"\x7f", raw"\x80", raw"\xff", raw"\xfe", raw"\x{7f}", raw"\x{0008f}")
```

Comment:
Can you add more unit test for larger hex digits (something close to `Character.MAX_CODE_POINT`) using the `\x{...}` syntax?